src/shared/components/event-line/event-line.component.html

```diff
@@ -0,0 +1,3 @@
+<span class="my-event-line">
+  <p class="my-event-text">Мое събитие</p>
```

Comment:
Make this dynamic with input. It can be used later in the admin with other text.